MobileSDK/sdk/src/main/java/org/lantern/mobilesdk/embedded/EmbeddedLantern.java

```diff
@@ -0,0 +1,30 @@
+package org.lantern.mobilesdk.embedded;
+
+import android.content.Context;
+
+import org.lantern.mobilesdk.Lantern;
+import org.lantern.mobilesdk.LanternNotRunningException;
+import org.lantern.mobilesdk.StartResult;
+
+public class EmbeddedLantern extends Lantern {
+    static {
+        // Track extra info about Android for logging to Loggly.
+        EmbeddedLantern.addLoggingMetadata("androidDevice", android.os.Build.DEVICE);
+        org.lantern.mobilesdk.Lantern.addLoggingMetadata("androidModel", android.os.Build.MODEL);
+        org.lantern.mobilesdk.Lantern.addLoggingMetadata("androidSdkVersion", "" + android.os.Build.VERSION.SDK_INT + " (" + android.os.Build.VERSION.RELEASE + ")");
```

Comment:
I think it makes sense to log to Loggly so that we get feedback on any issues that might be encountered by users of the embedded version.